integration/src/main/java/scripts/testscripts/ImportDataCollection.java

```diff
@@ -397,7 +397,7 @@ gain region from data collection. Workspace (policy=empty) + Data Collection (po
      Scenario 9: Group policy merging. WS(groupA) can merge DC(nogroup).
     */
     CreatedWorkspace groupTestWorkspace =
-        createWorkspaceWithRegionPolicy(workspaceApi, usaLocation);
```

Comment:
i don't follow why we need to remove the region policy? it shouldn't affect the group-constraint policy.

Reply:
It doesn't necessarily need to be removed. But all these scenarios were written with the idea that the region policy wasn't there. It seemed more straightforward to remove it rather than go back to adjust the comments and validation steps to accommodate it.

Reply:
sure